proto/events/events.proto

```
@@ -0,0 +1,32 @@
+syntax = "proto3";
```

Buf (1.55.1)

Files must have a package defined.

(PACKAGE_DEFINED)